stake-pool/program/src/processor.rs

```diff
@@ -819,6 +838,14 @@ impl Processor {
         // Stake program id
         let stake_program_info = next_account_info(account_info_iter)?;
 
+        // Check program ids
+        if *stake_history_info.key != solana_program::sysvar::stake_history::id() {
```

Comment:
nit just for your information :smile: This is where things get tricky sometimes -- since you'll just be passing this into the stake program, you can trust that it will check the sysvar id.   This check is not technically necessary, but it can't hurt (except for the computation budget)

Reply:
Right, I was aware of this because I did not add Rent or Clock checks :)

Reply:
So my mistake, removing this one.